source/MaterialXCore/Element.h

```diff
@@ -117,6 +117,14 @@ class Element : public std::enable_shared_from_this<Element>
     ///    the returned path will be relative to this ancestor.
     string getNamePath(ConstElementPtr relativeTo = nullptr) const;
 
+    /// Return the element referred to by the hierarchical path relative
```

Comment:
Perhaps this comment should be updated as well, so that it reads "return the descendant..."?

Reply:
Ok